django_startr/management/commands/startr.py

```
@@ -60,6 +64,20 @@ def get_selected_models(self, app, model_names):
             try:
                 return [app.get_model(model_name) for model_name in model_names]
             except:
-                raise CommandError("One or more of the models you entered for %s are incorrect." % app_label)
+                raise CommandError("One or more of the models you entered for %s are incorrect." % app.label)
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)